CHANGES.md

```diff
@@ -13,7 +13,20 @@ Format of the entries must be.
 * Entry two with no-newlines. (DCOS_OSS_JIRA_2)
 ```
 
-## DC/OS 1.12.3 (Next Release. Please Update News Entries in this Section).
+## DC/OS 1.12.4 (Next release, please update entries in this section)
+
+### Notable changes
```

Comment:
Hey @orsenthil @catsouthard I thought we do this kind of change to CHANGES.md as part of "cutting a release" process. Will this be added to some TODO list that is to be checked of every release we cut?

Reply:
Related: https://github.com/dcos/dcos/pull/3486#issuecomment-424670155